style/template.tex

```diff
@@ -60,15 +53,42 @@
 $if(geometry)$
 \usepackage[$for(geometry)$$geometry$$sep$,$endfor$]{geometry}
 $endif$
+\ifxetex
+  \usepackage[setpagesize=false, % page size defined by xetex
+              unicode=false, % unicode breaks when used with xetex
+              xetex]{hyperref}
+\else
+  \usepackage[unicode=true]{hyperref}
+\fi
+\usepackage[usenames,dvipsnames]{color}
```

Comment:
I think this is what's causing the build issue; it's from the default Pandoc template. I haven't yet figured out why this is a problem, given that it works locally.